### ISSUE TITLE
Search page does nothing without at least one "top" annotation present on default table.

### ISSUE THREAD

**Issue by mikedarcy** [informatics-isi-edu/chaise#387]
I loaded up a new catalog instance into ERMrest and configured Chaise to point at it.  After logging in, the default search page just sat there with the spinner spinning forever, appearing to "hang".  In reality, the app just got as far as schema introspection for the default table and then just stopped executing the rest of the page flow.

As soon as I put at least one "top" annotation on a column in my default table, the page rendered correctly.  I found this quirk particularly nasty, since I spent quite a bit of time trying to debug it.  This does not generate any errors in the console; things just simply stop working.

It would be nice if things just worked "out-of-the-box" against data that has not yet been annotated.  I am pretty sure this used to work properly.  The code should maybe just pick a column at random so this can be avoided.  Better still, maybe just choose the PK column as the default top column so at least the flow can continue.  I'm really not sure what the best solution is, but anything is better than the "hang" behavior I am seeing now.

**Comment by mikedarcy** [comment]
This seems to actually be a general problem with the search page display of any entity where no "top" columns exist.  The application should not be relying on the presence of the "top" annotation in order to  trigger the execution of a query and the subsequent display of the results, but that looks like what is happening.

**Comment by svoinea** [comment]
Fixed in commit 5f83c208275a42eabd54748bc6cfef7af0151f1f.

While the documentation https://github.com/informatics-isi-edu/chaise/blob/master/doc/configuration.md was already specifying for `facetPolicy: on_demand`: 
```
At start up, the facets with the "top" annotation will be selected.
```
the ever spinner when no top facet was defined, was a bug.

**Comment by mikedarcy** [comment]
Confirmed fixed.
